_data/domains.yml

```diff
@@ -184,11 +184,12 @@ websites:
       doc: http://help.eurodns.com/customer/en/portal/articles/1633106-how-do-i-activate-two-step-verification-on-my-account-
 
     - name: Fabulous
-      url: http://fabulous.com
+      url: https://fabulous.com
       img: fabulous.png
       tfa: Yes
-      hardware: yes
-      doc: http://fabulous.com/informationcenter/index.htm?formdata%5Bqid%5D=1194
+      hardware: Yes
+      software: Yes
+      doc: https://fabulous.com/knowledge/article/137:how+do+i+set+up+two-factor+authentication%3F
```

Comment:
https://fabulous.com/knowledge/article/137 seems to be enough.

Reply:
perfect! you can make the change!